package.json

```diff
@@ -115,7 +121,7 @@
     "eslint-plugin-jest": "22.15.2",
     "eslint-plugin-prettier": "3.1.0",
     "eslint-plugin-react": "7.14.3",
-    "inquirer": "6.5.2",
+    "inquirer": "7.0.0",
```

Comment:
should be moved to `dependencies`, not sure why it was already explicitly added

Reply:
It's being used in `publish.js`.